editor.planx.uk/src/@planx/components/FileUploadAndLabel/Public/index.test.tsx

```diff
@@ -396,26 +395,23 @@ describe("Adding tags and syncing state", () => {
       "file-tagging-dialog",
     );
 
-    // The number of autocompletes in the modal matches the number of uploaded files
-    const autocompleteInputs = getAllByRole("combobox", {
+    // The number of file cards in the modal matches the number of uploaded files
+    const uploadedFileCards = getAllByRole("heading", {
       name: /What does this file show/,
     });
-    expect(autocompleteInputs).toHaveLength(1);
+    expect(uploadedFileCards).toHaveLength(1);
 
     const submitModalButton = getByRole("button", { name: /Done/ });
     expect(submitModalButton).toBeVisible();
 
-    // Autocomplete is close, option unavailable
-    let roofPlanOption = queryByRole("option", { name: /Roof plan/ });
-    expect(roofPlanOption).toBeNull();
-
-    // Open dropdown
-    await user.click(autocompleteInputs[0]);
-    roofPlanOption = getByRole("option", { name: /Roof plan/ });
-    expect(roofPlanOption).toBeVisible();
+    // Find the "Roof plan" checkbox using data-testid which includes the file ID
+    // The pattern is: ${fileId}-${category}-${optionName}
+    const roofPlanCheckbox =
+      within(fileTaggingModal).getByTestId(/.*-.*-Roof-plan$/);
```

Comment:
I think we should be able to get these by label


```suggestion
    const roofPlanCheckbox =
      within(fileTaggingModal).getByLabelText("Roof plan");
```

Reply:
Yep, label text works and is semantically cleaner